### PR TITLE
Prepare for v3.2.0 release

### PR DESCRIPTION
**Pull request by meatballhat**
## What is the problem that this PR is trying to fix?

The size of [this delta](https://github.com/travis-ci/worker/compare/v3.1.0...HEAD).

## What approach did you choose and why?

Cut a release!  I chose v3.2.0 given the added features and lack of breaking public API changes.

## How can you test this?

👀

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,17 +30,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
     - account for transitional states when handling state update conflicts
     - delete self under various error conditions indicative of a requeue
```
Comment (bnferguson) [inline CHANGELOG.md]: Do we need a "Changed" for the SIGWINCH -> SIGUSR2 change? https://github.com/travis-ci/worker/commit/3ba889bb57e4953e5b5b08e96faec05fdefd90bf

**Comment by Lyoness** [comment]
:heart_decoration: the [delta of commits ](https://github.com/travis-ci/worker/compare/v3.1.0...HEAD) you just linked above... why not add that too in the Changelog at the end, for those who want moar exploar?

**Comment by meatballhat** [comment]
@Lyoness It's there! 😺 See https://github.com/travis-ci/worker/blob/master/CHANGELOG.md#unreleased

**Comment by Lyoness** [comment]
:picard-facepalm:  :shipit: :+1:

**Pull request merged**